namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-data-compliance-preprod/01-rbac.yaml

```diff
@@ -0,0 +1,14 @@
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: prison-data-compliance-preprod-admin
+  namespace: prison-data-compliance-preprod
+subjects:
+  - kind: Group
+    name: "github:dps-tech"
+    apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: ClusterRole
```

Comment:
Can you please create a role, similar to below, instead of using `ClusterRole`

https://github.com/ministryofjustice/cloud-platform-environments/blob/master/namespaces/live-1.cloud-platform.service.justice.gov.uk/c100-application-production/serviceaccount-circleci.yaml#L8